plugins/Denizen/scripts/jumpsplat/item_linker/chat_linker.dsc

```diff
@@ -0,0 +1,22 @@
+# Messages that contain {main_hand}, {off_hand}, {boots},
+# {leggings}, {chestplate} or {helmet} will instead replace
+# the bracketed text with the item in the corresponding slot.
+# Show off your cool stuff to all your friends!
+
+chat_linker:
+    type: world
+    debug: false
+    events:
+        on player chats priority:1000:
+            - define message <context.full_text>
+            - definemap items:
+                main_hand: <player.item_in_hand>
+                off_hand: <player.item_in_offhand>
+                boots: <player.equipment.get[1]>
+                leggings: <player.equipment.get[2]>
+                chestplate: <player.equipment.get[3]>
+                helmet: <player.equipment.get[4]>
+            - foreach <[items]> as:item:
+                - define display <[item].display.if_null[<[item].material.translated_name>].on_hover[<[item]>].type[SHOW_ITEM]>
+                - define message <[message].replace_text[{<[key]>}].with[<[display].underline>]>
+            - determine <[message]>
```

Comment:
How will this look on discord?

Reply:
Janky, it'll look like Denizen text formatting, so we have to account for that. Like `§[hover=SHOW_ITEM;i@wooden_sword]§[translate=item.minecraft.wooden_sword]§[/hover]` How are we currently handling Denizen text formatting?

Reply:
That’s a great question for @acikek

Reply:
We're not handling it at all. The raw message gets sent to the bridge channel.

Reply:
Well, this is a good point to bring up the `task` idea we were talking about before. Along with the task passing in all the contexts, we could also have the task pass in a `is_bridge` boolean, and then allow the task to handle it separately. So, for the chat linker example, if `is_bridge` is false, then it basically function as written, and `is_bridge` is true, then it maybe replaces the item with only the display name or something, rather than doing the full `SHOW_ITEM`.

Reply:
Just add `strip_color` to the chat bridge, it'll translate translatables, remove hovers, etc.